Add custom_tags variable in AWS network module (#91)
scalar-labs/scalar-terraform: docs/CustomTags.md

> # Adding Tags to Resources

Reviewer: ```suggestion
# Tagging Resources
```

> scalar-terraform provides a feature to add arbitrary tags to resources created.

Reviewer: ```suggestion
scalar-terraform provides a feature to add arbitrary tags to resources. The feature may be useful to group resources  so that you can do some operations for a particular group such as calculating cost.
```

I think the following explanations can be deleted.

Sometimes it is useful to add tags to cloud resources.

* Label metadata such as the owner and the environment
* Organize billing to summerize the system-wide costs

> At present, this feature only supports AWS.

Reviewer: ```suggestion
At present, the feature is only supported in AWS.
```

> ## Setting Tags

Reviewer: ```suggestion
## How to add tags
```

> In example.tfvars file in `network` module, you can add as many tags as needed in the `custom_tags` variable.

Reviewer: It sounds misleading since it can be used in non-example projects.
```suggestion
You can use `custom_tags` variable defined in `network` module to add tags to resources.
```

> The tags specified here will be applied to all resources created by other modules that reference this module's output.

Reviewer: ```suggestion
The tags will be applied to all the resources created by other modules that reference `network` module's output.
```

```
custom_tags = {
  Environment = "production"
  Owner       = "scalar"
}
```

> Note that the following tag keys are reserved by the tool and could be ignored even if you set them in `custom_tags`.

Reviewer: ```suggestion
Note that the following tag keys are reserved by the tool so they are ignored if they are set in `custom_tags`.
```
